inventory/sample/hosts.ini

```diff
@@ -1,11 +1,12 @@
 # ## Configure 'ip' variable to bind kubernetes services on a
 # ## different ip than the default iface
-# node1 ansible_host=95.54.0.12  # ip=10.3.0.1
-# node2 ansible_host=95.54.0.13  # ip=10.3.0.2
-# node3 ansible_host=95.54.0.14  # ip=10.3.0.3
-# node4 ansible_host=95.54.0.15  # ip=10.3.0.4
-# node5 ansible_host=95.54.0.16  # ip=10.3.0.5
-# node6 ansible_host=95.54.0.17  # ip=10.3.0.6
+# ## We should set etcd_member_name for etcd cluster. The node that is not a etcd member do not need to set the value, or can set the empty string value.
+# node1 ansible_host=95.54.0.12  # ip=10.3.0.1 etcd_member_name=etcd1
```

Comment:
Why would you do this, and not use the hostname itself? Seems unlogic?

Reply:
@woopstar ,
Please see discuss above,
This will happen in the following cases:
 (1)When the order of the etcd nodes number name in the front and back deployment is different, the etcd_member_name, etcd_peer_addresses and etcd_events_peers_addresses are different, so that etcd cluster is failed.
 (2)When we want to change etcd member, we can't execute this deployment again to get the right result. e.g. we need to recover master and etcd cluster in my issue #2936 , Recover master cluster .(Please review it)

We need etcd_member_name. We can use etcdn or hostname, and etcdn is better.

Reply:
I see the discussion, but why not just use the ansible hostname, instead of predefining the name? Then the order does not matter either?

Reply:
I have discuss with @Atoms , but discuss content may be lost after modified the pr.

Reply:
inventory_hostname and etcdn are ok, but the original etcd_member_name is etcdn. I only resolve order bug. If we use inventory_hostname, then etcd member names are node1, node2,...., this does not show etcd feature.
If you want me modified pr, I can do it.